### PR TITLE
UICIRC-955: Add additional fields to reminder fees

### PR DESCRIPTION
**Pull request by mkuklis**
https://issues.folio.org/browse/UICIRC-955

Add additional fields to the reminder fees section.

### CODE REVIEW

File: src/settings/FinePolicy/components/EditSections/ReminderFeesSection/ReminderFeesSection.js [hunk 1]
```diff
@@ -52,6 +52,28 @@ const ReminderFeesSection = ({ sectionOpen, noticeTemplates, blockTemplates }) =
             />
           </Col>
         </Row>
+        <Row>
+          <Col xs={3}>
+            <Field
+              data-testid="ignoreGracePeriodHoldsTestId"
+              label={<FormattedMessage id="ui-circulation.settings.finePolicy.reminderFees.ignoreGracePeriodHolds" />}
+              name="reminderFeesPolicy.ignoreGracePeriodHolds"
+              dataOptions={dataOptions}
```
Comment (MikeTaylor) [inline src/settings/FinePolicy/components/EditSections/ReminderFeesSection/ReminderFeesSection.js:61]: As a tiny tiny matter of style, I prefer to say `component={Select}` _before_ `dataOptions={dataOptions}`, as the data-options are a refinement of how the selector works. Feel free to ignore this if you don't agree, or if consistent style in this code is to use the order you went with here.

File: src/settings/Models/ReminderFeesPolicy/ReminderFeesPolicy.test.js [hunk 2]
```diff
@@ -23,12 +23,26 @@ describe('ReminderFeesPolicy', () => {
 
   it('"defaultReminderFeesPolicy" should return correct data', () => {
     const expectedPolicy = {
-      countClosed: true,
-      ignoreGracePeriodRecall: true,
-      clearPatronBlockWhenPaid: true,
+      countClosed: false,
```
Comment (MikeTaylor) [inline src/settings/Models/ReminderFeesPolicy/ReminderFeesPolicy.test.js:26]: Is it deliberate that the values provided for some of the existing field have changed?

Comment (mkuklis) [reply src/settings/Models/ReminderFeesPolicy/ReminderFeesPolicy.test.js:26]: Yes @MikeTaylor, the previous implementation was buggy. What is coming by default from the server is `false` so we just need to make sure we align with that here.

File: src/settings/FinePolicy/components/EditSections/ReminderFeesSection/ReminderFeesSection.js (hunk 1 above)
Comment (mkuklis) [reply src/settings/FinePolicy/components/EditSections/ReminderFeesSection/ReminderFeesSection.js:61]: good idea @MikeTaylor!

**Pull request merged**